authorization-server/jwt-authorization-grant-token-exchange.js

```diff
@@ -4,8 +4,18 @@ import validatePresence from 'oidc-provider/lib/helpers/validate_presence.js';
 import instance from 'oidc-provider/lib/helpers/weak_cache.js';
 import { OAuthBadRequest, requestIdJwtAuthzGrant } from 'id-assert-authz-grant-client';
 import { getIdToken } from './utils/id-token-cache.js';
-
+import createRedisClient from './utils/redis.js';
 // eslint-disable-next-line import/prefer-default-export
+const saveTokenToRedis = async (token) => {
+  try {
+    const redisClient = await createRedisClient();
+    await redisClient.connect();
```

Comment:
i forget, but is this the correct redis pattern?  We connect every time we save a token?  I would expect a single redis connection to be established at server startup